site/en/docs/extensions/mv3/messaging/index.md

```diff
@@ -265,7 +271,7 @@ scripts.
 When receiving a message from a content script or another extension, your scripts should be careful
 not to fall victim to [cross-site scripting][39]. This advice applies to scripts running inside the
 extension background page as well as to content scripts running inside other web origins.
-Specifically, avoid using dangerous APIs such as the ones below:
+Specifically, avoid using dangerous APIs, specificaslly `chrome.scripting.executeScript()`, `eval()`, and `new Function()`. For example:
```

Comment:
This section is a bit odd now because of CSP changes in Manifest V3. Ian is working on #3837 to update our CSP docs, but the short version is that Chrome has a minimum CSP for extension contexts that does not allow developers to call `eval()` or `new Function()`. For reference, the minimum CSP is `script-src 'self' 'wasm-unsafe-eval'; object-src 'self';`.

It may be best to use a slightly different approach for this section. Since cross-site scripting isn't as much a concern, we could lean into "cross-site attacks," where a compromised render could use the message channel to send the extension messages in order to extract data or make the extension do something dangerous.

Cross-site scripting is still a concern for content scripts. This is another method by which a content script could potentially be hijacked into sending dangerous messages back to the extension.

Reply:
I'm going to remove the new text so that we can merge this. I'll circle back when I have more information.